testsuite/tests/singlecluster/gateway/reconciliation/listeners/conftest.py

```diff
@@ -0,0 +1,65 @@
+"""
+Conftest for Gateway listeners tests.
+The main change consists of replacing the default wildcard domain for an exact one.
+"""
+
+import pytest
+
+from testsuite.gateway.gateway_api.hostname import StaticHostname
+
+
+@pytest.fixture(scope="module")
+def domain_prefix():
+    """Prefix of default domain."""
+    return "prefix"
```

Comment:
why do you need these custom prefixes?

Reply:
Just so each test has unique prefix in domain name for easier debugging. This is maybe redundant feature

Reply:
you can just simply do: `f'{blame("prefix1")}.{base_domain}'` Don't need to define fixture

Reply:
Done :+1: